Update fuzziness in imported/ for accelerated drawing

nmoucht opened the pull request:
#### 3b72ff8da60c846bd356cadbec7f05b5b3e79259
<pre>
Update fuzziness in imported/ for accelerated drawing
<a href="https://bugs.webkit.org/show_bug.cgi?id=246895">https://bugs.webkit.org/show_bug.cgi?id=246895</a>
rdar://101450189

Reviewed by Simon Fraser.

Update fuzziness in imported/ for when accelerated drawing is turned on by default.

* LayoutTests/imported/blink/compositing/will-change/stacking-context-creation.html:
* LayoutTests/imported/blink/css3/blending/mix-blend-mode-with-squashing-layer.html:
* LayoutTests/imported/mozilla/svg/linearGradient-basic-03.svg:
* LayoutTests/imported/mozilla/svg/marker-orientation-01.svg:
* LayoutTests/imported/mozilla/svg/opacity-and-gradient-02.svg:
* LayoutTests/imported/mozilla/svg/rect-02.svg:
* LayoutTests/imported/w3c/web-platform-tests/css/css-backgrounds/background-repeat-space-6.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-backgrounds/background-size-025.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-backgrounds/background-size-027.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-contain/content-visibility/content-visibility-012.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-contain/content-visibility/content-visibility-014.html:

Canonical link: <a href="https://commits.webkit.org/255978@main">https://commits.webkit.org/255978@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/39253fa1ec349659bb7b27f5830100c27e8494ad

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/6/builds/94233 "Passed style check") | [  ~~🛠 ios~~](https://ews-build.webkit.org/#/builders/77/builds/3421 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 mac~~](https://ews-build.webkit.org/#/builders/43/builds/25126 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 wpe~~](https://ews-build.webkit.org/#/builders/8/builds/103911 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 🧪 win~~](https://ews-build.webkit.org/#/builders/10/builds/164189 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") 
| | [  ~~🛠 ios-sim~~](https://ews-build.webkit.org/#/builders/76/builds/3450 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 mac-debug~~](https://ews-build.webkit.org/#/builders/71/builds/31635 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 gtk~~](https://ews-build.webkit.org/#/builders/36/builds/86585 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 wincairo~~](https://ews-build.webkit.org/#/builders/12/builds/99932 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/19/builds/99889 "Passed tests") | [  ~~🧪 ios-wk2~~](https://ews-build.webkit.org/#/builders/76/builds/3450 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🛠 mac-AS-debug~~](https://ews-build.webkit.org/#/builders/61/builds/80641 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/36/builds/86585 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [  ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/76/builds/3450 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-mac~~](https://ews-build.webkit.org/#/builders/43/builds/25126 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-gtk~~](https://ews-build.webkit.org/#/builders/36/builds/86585 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [  ~~🛠 tv~~](https://ews-build.webkit.org/#/builders/66/builds/38018 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-wk1~~](https://ews-build.webkit.org/#/builders/43/builds/25126 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/67/builds/35903 "Built successfully") | [  ~~🧪 mac-wk2~~](https://ews-build.webkit.org/#/builders/43/builds/25126 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/63/builds/39782 "Built successfully") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/61/builds/80641 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/75/builds/1946 "Built successfully and passed tests") | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/65/builds/41726 "Built successfully") | [  ~~🧪 mac-wk2-stress~~](https://ews-build.webkit.org/#/builders/43/builds/25126 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
<!--EWS-Status-Bubble-End-->

JonWBedard commented on Tools/Scripts/webkitpy/layout_tests/run_webkit_tests.py:
```diff
@@ -117,7 +117,7 @@ def parse_args(args):
         optparse.make_option("--complex-text", action="store_true", default=False,
             help="Use the complex text code path for all text (OS X and Windows only)"),
-        optparse.make_option("--accelerated-drawing", action="store_true", default=False,
+        optparse.make_option("--accelerated-drawing", action="store_true", default=True,
```
This feels like a big change worthy of a separate PR (and no doubt, the reason for `webkitpy` test failures.

nmoucht replied:
Yeah this is going to be enabled here: https://bugs.webkit.org/show_bug.cgi?id=246285. I wanted to make sure these updates cause the tests to pass with this flag enabled. I'll remove it once I get the results.

JonWBedard replied:
Makes sense!

nmoucht committed:
Update fuzziness in imported/ for accelerated drawing
https://bugs.webkit.org/show_bug.cgi?id=246895
rdar://101450189

Reviewed by Simon Fraser.

Update fuzziness in imported/ for when accelerated drawing is turned on by default.

* LayoutTests/imported/blink/compositing/will-change/stacking-context-creation.html:
* LayoutTests/imported/blink/css3/blending/mix-blend-mode-with-squashing-layer.html:
* LayoutTests/imported/mozilla/svg/linearGradient-basic-03.svg:
* LayoutTests/imported/mozilla/svg/marker-orientation-01.svg:
* LayoutTests/imported/mozilla/svg/opacity-and-gradient-02.svg:
* LayoutTests/imported/mozilla/svg/rect-02.svg:
* LayoutTests/imported/w3c/web-platform-tests/css/css-backgrounds/background-repeat-space-6.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-backgrounds/background-size-025.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-backgrounds/background-size-027.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-contain/content-visibility/content-visibility-012.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-contain/content-visibility/content-visibility-014.html:

Canonical link: https://commits.webkit.org/255978@main
